src/server_manager/install_scripts/install_server.sh

```diff
@@ -235,11 +268,17 @@ install_shadowbox() {
   # require new dependencies.
   cat <<END_OF_SERVER_OUTPUT
 
-Please copy the following configuration to your Outline Manager:
+CONGRATULATIONS! Your Outline Server is up and running.
```

Comment:
Outline Server -> Outline server

(and throughout)

Reply:
Done